drivers/sensor/max30101/Kconfig

```diff
@@ -13,6 +13,8 @@ menuconfig MAX30101
 
 if MAX30101
 
+if !HAS_DTS_I2C_DEVICE
+
```

Comment:
I can't comment on the proper line, but can you remove the !HAS_DTS_I2C, now that this is wrapped in a !HAS_DTS_I2C_DEVICE